src/components/Home/HomeSchemaOrg.jsx

```diff
@@ -1,30 +1,56 @@
 import { useIntl } from 'react-intl';
 import { toPublicURL } from '@plone/volto/helpers';
+import { useSelector } from 'react-redux';
 import { SiteProperty } from 'volto-site-settings';
 import { SchemaOrg, getSiteProperty } from 'io-sanita-theme/helpers';
 
 const HomeSchemaOrg = ({ content }) => {
   const intl = useIntl();
+  const socialSettings = useSelector((state) => state.socialSettings?.results);
+
+  const name = SiteProperty({
+    property: 'site_title',
+    defaultValue: getSiteProperty('siteTitle', intl.locale),
+    getValue: true,
+  });
+  const description = SiteProperty({
+    property: 'site_subtitle',
+    defaultValue: getSiteProperty('siteSubtitle', intl.locale),
+    getValue: true,
+  });
+
   let schemaOrg = null;
   if (content?.['@type'] == 'Plone Site' || content?.['@type'] == 'LRF') {
-    const name = SiteProperty({
-      property: 'site_title',
-      defaultValue: getSiteProperty('siteTitle', intl.locale),
-      getValue: true,
-    });
-    const description = SiteProperty({
-      property: 'site_subtitle',
-      defaultValue: getSiteProperty('siteSubtitle', intl.locale),
-      getValue: true,
-    });
+    let socialLinks = null;
+    if (socialSettings && socialSettings?.length > 0) {
+      socialLinks = socialSettings.map((item) => item.url);
+    }
 
     schemaOrg = {
-      '@type': 'MedicalOrganization',
-      name: name,
-      description: description,
-      url: toPublicURL(content['@id']),
+      '@graph': [
+        {
+          '@type': 'WebSite',
+          url: toPublicURL(content['@id']),
```

Comment:
occhio che se è multilingua qui ci viene scritto ad esempio https:/www.irst.it/it se si è in italiano, o
https:/www.irst.it/en se si è in inglese...

l'url del sito corretto è https:/www.irst.it 

```suggestion
          url: toPublicURL('/'),
```